relay-server/src/actors/outcome.rs

```diff
@@ -304,6 +306,8 @@ pub struct TrackRawOutcome {
     /// The source of the outcome (which Relay sent it)
     #[serde(default, skip_serializing_if = "Option::is_none")]
     source: Option<String>,
+    /// The data categories of the events
+    pub data_categories: Vec<DataCategory>,
```

Comment:
Please annotate this with `#[serde(default)]`. Usually, serde would require this new field and reject any past payload that we receive without. Since our PoP Relays are sending outcomes already without this, we need to gracefully accept them.

When going with a single DataCategory, you may have to specify an explicit default via `#[serde(default = "default_data_category")]` and then define a function:

```rust
fn default_data_category() -> DataCategory {
    DataCategory::Error
}
```

Reply:
Will do. But (if it's no longer a `Vec`), would `Option<DataCategory>` be okay? That seems a natural fit with `infer_event_category`.